doc/introduction/index.rst

```diff
@@ -87,6 +87,64 @@ platform to run both safety-critical applications and non-safety
 applications, together with security functions that safeguard the
 system.
 
+Industrial Workload Consolidation
+=================================
+
+.. figure:: images/ACRN-V2-industrial-scenario.png
+   :width: 600px
+   :align: center
+   :name: V2-industrial-scenario
+
+   ACRN Industrial Workload Consolidation scenario
```

Comment:
I just have open question, do we need to rename "industry" scenario in current code to "industrial"?

Reply:
I would prefer if we use `industrial` too (instead of `industry`).

Reply:
@gvancuts @wenlingz @jsun26intel the term "industry" is what's used in Kconfig and .xml configuration files

Reply:
We are not ready to change "industry" to "industrial" now, since it impacts CI and automation script, let's reschedule it in future.